main.go

```diff
@@ -52,7 +53,10 @@ func main() {
 	flag.Int("pprofPort", 0, "The port to use for the pprof http endpoints")
 	flag.String("web3", "https://mainnet.optimism.io", "Web3 RPC Optimism endpoint")
 
-	//flag.String("neynarSignerUUID", "", "neynar signer UUID")
+	// bot flags
+	flag.Uint64("botFid", 0, "FID to be used for the bot")
+	flag.String("botPrivKey", "", "The bot private key to use for signing the vote (hex)")
+	flag.String("hubEndpoint", "", "The hub endpoint to use")
```

Comment:
As far as I understand, the current implementation only runs the bot if there is a hubEndpoint defined. That means all neynar webhooks logic is currently useless, isn't it? 

That's ok if its a temporary measure, but you must add a big notice in the code explaining why. Also you should explicitly say on the flags that for the bot to run you need to define a hubEndpoint (also maybe renaming it to botHubEndpoint)